bot/cogs/antispam.py

```diff
@@ -245,16 +246,17 @@ def mod_log(self) -> ModLog:
         await deletion_context.upload_messages(self.bot.user.id, self.mod_log)
 
 
-def validate_config() -> bool:
+def validate_config(rules: Mapping = AntiSpamConfig.rules) -> dict:
```

Comment:
Since we map strings to strings, this could be:
```suggestion
def validate_config(rules: Mapping = AntiSpamConfig.rules) -> Dict[str, str]:
```